src/backend/joanie/core/serializers.py

```diff
@@ -346,22 +346,41 @@ class EnrollmentSerializer(serializers.ModelSerializer):
     course_run = serializers.SlugRelatedField(
         queryset=models.CourseRun.objects.all(), slug_field="resource_link"
     )
-    order = serializers.SlugRelatedField(
-        queryset=models.Order.objects.all(), slug_field="uid", required=False
+    orders = serializers.SlugRelatedField(
+        queryset=models.Order.objects.all(), slug_field="uid", required=False, many=True
     )
 
     class Meta:
         model = models.Enrollment
-        fields = ["id", "user", "course_run", "order", "is_active", "state"]
+        fields = ["id", "user", "course_run", "orders", "is_active", "state"]
         read_only_fields = ["id", "user", "state"]
 
+    def create(self, validated_data):
+        """
+        Check if enrollment for the couple user/course run does not already exist.
```

Comment:
it sucks that this is called "create" but can do an "update". Is it within reach to refactor this design?

Reply:
but there would still be a method called "create" which tries to update something instead of creating it? So I think the create method is not well named.